src/reactive-membrane.ts

```diff
@@ -53,12 +53,23 @@ function defaultValueIsObservable(value: any): boolean {
     if (value == null) {
```

Comment:
If we are now speaking about micro-optimization. We should also optimize this branch. The `==` operator makes a type cohersion. With @caridy proposed below you can change the condition to:

```suggestion
    if (value === null) {
```

The case where `value` is `undefined` will be covered by `typeof value` returning `
"undefined"`.